docs/android/external-native-dependencies.md

```diff
@@ -1,34 +1,33 @@
 # Add a new external native dependency
 
-When we update a dependency there are no major changes as versions will be taken from [.build/package.json](../../.build/package.json) and a new [snapshot](http://trinerdis.cz:8000/repository/snapshots/com/trinerdis/skypicker/) will be deployed.
+When we update a dependency there are no major changes as versions will be taken from 
+[.build/package.json](../../.build/package.json) and a new 
+[snapshot](http://trinerdis.cz:8000/repository/snapshots/com/trinerdis/skypicker/) will be deployed.
 
-To add a new native dependency though, that is more time consuming. We need to create a separated `SNAPSHOT` for it and upload it to [Trinerdis](http://trinerdis.cz:8000/repository/snapshots/). We need to create a similar structure for a new dependency like in [android/native/react-native-tooltips](../../android/native/react-native-tooltips).
+To add a new native dependency though, that is more time consuming. We need to modify the 
+`[android/build.gradle](../../android/build.gradle)`.
+This will deploy a new `SNAPSHOT` for it and upload it to 
+[Gitlab packages](https://gitlab.skypicker.com/mobile/android/-/packages).
 
-Usually we only need to change the following part to use the correct dependency name:
+We need to add a new case on the [gradle switch-case](../../android/build.gradle#L77) similar to:
```

Comment:
In future this can be improved further so there will be default branch (expecting consistent naming of artifacts and version variables) and only exceptions will be handled in separate cases.